spellbook/models/dex/dex_schema.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+
+models:
+  - name: dex_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      contributors: jeff-dude
+    config:
+      tags: ['ethereum','uniswap','trades', 'dex','cross-chain']
+    description: >
+        DEX trades on all chains across all contracts and versions
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
```

Comment:
Setting a very high column description standard :) 

I do feel like maybe this is a bit obvious and although its a significant improvement on
description: "Blockchain" 
maybe we don't need a description at all. 

It may make writing schemas less intimidating if we reserve descriptions for ambiguous columns. 

But that does mean we have to guess which columns aren't clear to users.

Reply:
agree on all points here. i wanted to put it out there for users to know it's an option. happy to hear from potentially @dsalv @aalan3 or @0xBoxer on this. 

keep all column descriptions, then clean up wording on each where applicable?
remove descriptions from the obvious columns (help determine what's obvious)?